react/week1/my-app/src/App.js

```diff
@@ -0,0 +1,33 @@
+//import logo from './logo.svg';
+import './App.css';
+import './ToDoItem.css';
+
+
+
+import ToDoItem from './ToDoItem.js';
+import ToDoList from './ToDoList.js';
+
+
+function App() {
+
+  return (
+
+    <div className="App">
+      <header className="App-header">
+        <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTIJoynbf7Cqar_m8fr8VWdKIVnDT-oampspA&usqp=CAU" alt=" girl with list" className="App-logo" />
```

Comment:
I couldn't run the code but couldn't help but take a look at what this image was - nice touch!

Reply:
Hi, I could run the code on local host using create-react-app already.can you please go through the code again to run it.
thanks for your feedback!